Model/Cron.php

```diff
@@ -1527,38 +1527,27 @@ protected function _refundOrder()
     protected function _authorizePayment()
     {
         $this->_adyenLogger->addAdyenNotificationCronjob('Authorisation of the order');
+        $this->isAutoCapture = $this->_isAutoCapture();
 
         // Set adyen_notification_event_code_success to true so that we ignore a possible OFFER_CLOSED
         if (strcmp($this->_success, 'true') == 0) {
             $this->_order->setData('adyen_notification_event_code_success', 1);
         }
-        $fraudManualReviewStatus = $this->getFraudManualReviewStatus();
-        $this->adyenOrderPaymentHelper->createAdyenOrderPayment($this->_order, $this->notification, $this->_isAutoCapture());
+
+        $this->adyenOrderPaymentHelper->createAdyenOrderPayment($this->_order, $this->notification, $this->isAutoCapture);
         $isTotalAmountAuthorised = $this->_isTotalAmount(
             $this->_order->getPayment()->getEntityId(),
             $this->orderCurrency
         );
 
-        // If manual review is NOT active OR no custom status is set on the Adyen config page
-        if (!$this->requireFraudManualReview || $fraudManualReviewStatus == "") {
-            if ($isTotalAmountAuthorised) {
-                $this->_setPrePaymentAuthorized();
-            } else {
-                $this->_order->addStatusHistoryComment(__(sprintf(
-                    'Partial authorisation w/amount %s %s was processed',
-                    $this->_currency,
-                    $this->_adyenHelper->originalAmount($this->_value, $this->_currency)
-                )), false);
-            }
+        if ($isTotalAmountAuthorised) {
+            $this->_setPrePaymentAuthorized();
```

Comment:
I added a small fix to this in this [commit](https://github.com/Adyen/adyen-magento2/pull/1274/commits/89c62c85f8d9dbd412dc299ee9028648808802c3). Although I'm aware that this if condition isn't the cleanest solution, this check will eventually be done better once we handle other PRs related to the refactoring of the `Cron.php` file, which will also include the functions you mentioned in the parent comment. This should be done in the near future.

Reply:
This could work but it is not really clean, especially if in the feature PR (or other opened PR), for some reason, someone tries to save the order after state is set to 'processing'.

I will update my local env with this to make other checks.  

I'll keep you updated,
Thanks

Reply:
Hi @mirkocesaro 

I agree that this is not the cleanest solution, but it is something that will be updated eventually. This PR is the first attempt to refactor the code related to case management and webhook handling. I'm sure that after this first iteration, we will have more PRs to clean up more related code and appropriately handle the edge cases. If no further issues are found, we plan to merge this PR to the develop branch by the end of this week.

In the meantime, we greatly appreciate all your feedback!

Jean,
Adyen